src/components/pages/home/Home.js

```diff
@@ -3,24 +3,88 @@ import React, { useState, useEffect } from 'react'
 import Products from '../../products/Products'
 import SearchBar from '../../searchbar/Searchbar'
 import itemsServices from '../../../services/items'
+import PaginationButton from '../../PaginationButtons/PaginationButton'
 
 import './home.css'
 
+const PAGE_SIZE = 6;
+
 const Home = () => {
     const [items, setItems] = useState([])
+    const [noItems, setNoItems] = useState(false)
+    const [number_of_pages, setNumberOfPages] = useState(null)
```

Comment:
This isn't necessary to have as a state variable. In fact, you can even move this logic to the Pagination component.